src/peergos/server/tests/simulation/AccessControl.java

```diff
@@ -0,0 +1,90 @@
+package peergos.server.tests.simulation;
+
+import java.nio.file.Path;
+import java.util.*;
+
+public interface AccessControl {
+
+    String getOwner(Path path);
+
+    List<String> get(Path path, FileSystem.Permission permission);
```

Comment:
Any reason why this is a list instead of a set?

Reply:
No, not fussed really.